nshandle_linux.go

```diff
@@ -39,9 +39,9 @@ func (ns NsHandle) String() string {
 	return fmt.Sprintf("NS(%d: %d, %d)", ns, s.Dev, s.Ino)
 }
 
-// UniqueId returns a string which uniquely identifies the namespace
+// UniqueID returns a string which uniquely identifies the namespace
 // associated with the network handle.
-func (ns NsHandle) UniqueId() string {
+func (ns NsHandle) UniqueID() string {
```

Comment:
This is a breaking change is it not?

In general I don't mind breaking changes like this where the type checker will complain, but we'd need to ensure to bump the major version.

And also, I do wonder, is it really worth it to make a breaking change purely for a cosmetic lint?